src/main/webapp/assets/feedStyle.css

```diff
@@ -44,10 +43,175 @@
   cursor: pointer;
 }
 
+#modal-background {
+  background: white;
+  background-color: rgba(0, 0, 0, 0.4);
+  display: none;
+  height: 100%;
+  position: fixed;
+  width: 100%;
+  top: 0;
+  z-index: 1;
+}
+
+#modal-create-post {
+  background: white;
+  border-radius: 15px;
+  bottom: 15%;
+  box-shadow: 0 1px 10px grey, 0 -1px 10px grey;
+  left: 25%;
+  max-height: 450px;
+  padding: 30px;
+  position: fixed;
+  width: 50%;
+  overflow: auto;
+  z-index: 1;
+  top: 15%;
+}
+
+#modal-close {
+  background: none;
+  border: none;
+  font-size: 1.25rem;
+  outline: none;
+  position: absolute;
+  right: 15px;
+  top: 15px;
+}
+
+#modal-title {
+  margin-left: 12px;
+}
+
+input[type=text] {
+  border: none;
+  border-radius: 50px;
+  font-size: 1rem;
+  margin-bottom: 0;
+  margin-top: 0;
+  outline: none;
+}
+
+input[type=number] {
+  background: rgba(27, 24, 24, 0.1);
```

Comment:
Nit: Google style asks for hex instead of rgba!

Reply:
Good catch